tests/test_client.py

```diff
@@ -26,8 +25,8 @@ def test_instantiate(index_client):
 
 
 def test_create_with_metadata(index_client):
-    urls = ['s3://bucket/key']
-    urls_metadata = {'s3://bucket/key': {'k': 'v'}}
+    urls = ['s3://s3-host.com/bucket/key']
```

Comment:
so all commons except GDC use the standard s3 uri format which is s3://bucket/key
So I think we can keep the example one as how it looks like before